xcm-transactor/src/lib.rs

```diff
@@ -75,35 +72,37 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
-		// This function should really only be called once via governance
-		// There is chance for it to be called another time or two in the future
-		// This weight is ARBITRARY.
+		/// Send swap instruction to the relay chain to swap the slot lease of our two parachains.
+		/// This needs to be done from within a pallet as the `XCM` origin must be the parachain
+		/// itself.
+		///
```

Comment:
[Style only]
I think that one empty line is enough.

Reply:
Haha, I will gladly do this for you!